corehq/apps/hqadmin/views/users.py

```diff
@@ -158,15 +158,26 @@ def _validate_user_access(self, user):
         return True
 
     def get(self, request, *args, **kwargs):
+        self.is_web_user = request.GET.get('webuser', False)
```

Comment:
To be in line with the other boolean vars in this code:
```suggestion
        self.is_web_user = request.GET.get('webuser') == 'true'
```

Reply:
Actually, you could get rid of this extra parameter entirely.  As long as there's a `@` in the username, it can be looked up, then you can call `couch_user.is_web_user()` or `couch_user.is_commcare_user()` as needed, without requiring the user to specify.

Reply:
Okay I tried to address this in the new commits with some checks and removed the `webuser` flag.